core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AutoscaleV4Controller.java

```diff
@@ -157,8 +157,7 @@ public ClusterProxyConfiguration getClusterProxyconfiguration() {
     }
 
     @Override
-    @DisableCheckPermissions
-    @PreAuthorize("hasRole('AUTOSCALE')")
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.DATAHUB_READ)
     public AutoscaleRecommendationV4Response getRecommendation(Long workspaceId, String blueprintName) {
```

Comment:
This api is meant for internal periscope use which uses internalCrn for communication. It should use the earlier PreAuthorize checks  and not CheckPermissionByAccount.

Reply:
That makes sense. How to test those internal APIs (in general)?